src/app/app.component.ts

```diff
@@ -56,6 +56,8 @@ export class App {
   }
 
   setNavState() {
+    const event = new CustomEvent('navExpand');
+    window.dispatchEvent(event);
     this.navOpen = !this.navOpen;
   }
 
```

Comment:
We might be better off using ionic events - https://ionicframework.com/docs/api/util/Events/